engines/grim/detection.cpp

```diff
@@ -56,6 +56,15 @@ static const ADExtraGuiOptionsMap gameGuiOptions[] = {
 			false
 		}
 	},
+	{
+		GAMEOPTION_LOAD_DATAUSR,
```

Comment:
Unfortunately that's the way engine options work. They are attached to detection entries.
May I suggest using defines so that the options are not duplicated all over the place?
It is sometimes done like that in ScummVM: https://github.com/scummvm/scummvm/blob/master/engines/mohawk/detection_tables.h#L30